ownership-chain/precompile/asset-metadata-extender/contracts/AssetMetadataExtender.sol

```diff
@@ -70,4 +70,22 @@ interface AssetMetadataExtender {
         string calldata uloc,
         uint32 index
     ) external view returns (string memory);
+
+    /// @notice Retrieves the extension of a token's universal location by a claimer
```

Comment:
minor rephrasing: "Retrieves the extension of a universal location made by the provided claimer"